src/views/main.tsx

```diff
@@ -1,9 +1,40 @@
+import { useContext } from "react";
 import MapAP from "../components/MapAP";
+import isLoggedInContext from "../context/context";
+import "../context/context.css";
+import Mountains from "../components/mountains";
 
 export default function Success() {
+  const context = useContext(isLoggedInContext);
+
+  if (!context) {
+    throw new Error("useContext must be used within an IsLoggedInProvider");
+  }
+
+  const { loggedIn } = context;
+
   return (
     <>
-      <MapAP />
+      {loggedIn ? (
+        <MapAP />
+      ) : (
+        <div className="main bg">
```

Comment:
Can we extract this code into a separate component? Maybe we will have to use it again somewhere else.